tests/name_constraints/generate.py

```diff
@@ -0,0 +1,282 @@
+from cryptography import x509
```

Comment:
I don't think there is anything comparable to https://cryptography.io/en/latest/x509/reference/#x-509-certificate-builder in the rust ecosystem for building certificates. Closest is probably [x509-certificate](https://docs.rs/x509-certificate/latest/x509_certificate/certificate/struct.X509CertificateBuilder.html) but it seems not to support name constraints, or subjectAltName. (Ironically these parts of cryptography.io are actually written in rust, but as an internal implementation detail rather than a reusable crate.)

Reply:
Also I don't think `cryptography.io` is a great canonical name for this to use in the commit message etc? I'd find `pyca/cryptography` easier to understand (purely because I happen to be familiar with it), or maybe specify something like "the widely used Python cryptography package".